air-script/tests/main.rs

```diff
@@ -1,86 +1,221 @@
-use expect_test::expect_file;
+use std::fs::File;
+use std::io::prelude::*;
 
+use expect_test::expect_file;
 mod helpers;
 use helpers::Test;
 
 // TESTS
 // ================================================================================================
 
 #[test]
-fn aux_trace() {
+fn winterfell_aux_trace() {
     let generated_air = Test::new("tests/aux_trace/aux_trace.air".to_string())
-        .transpile()
-        .unwrap();
+        .unwrap()
+        .generate_winterfell();
 
     let expected = expect_file!["aux_trace/aux_trace.rs"];
     expected.assert_eq(&generated_air);
 }
 
 #[test]
-fn binary() {
-    let generated_air = Test::new("tests/binary/binary.air".to_string())
-        .transpile()
+fn gce_aux_trace() {
+    Test::new("tests/aux_trace/aux_trace.air".to_string())
+        .unwrap()
+        .generate_gce(2, "tests/aux_trace/generated_aux_trace.json")
```

Comment:
Let's not unwrap twice in the same chain here. Instead, we can do something like this.

```
    let test = Test::new("tests/aux_trace/aux_trace.air".to_string()).unwrap();
    assert!(test
        .generate_gce(2, "tests/aux_trace/generated_aux_trace.json")
        .is_ok());
```